torch/distributed/rpc/api.py

```diff
@@ -572,6 +580,14 @@ def rpc_async(to, func, args=None, kwargs=None):
         args (tuple): the argument tuple for the ``func`` invocation.
         kwargs (dict): is a dictionary of keyword arguments for the ``func``
                        invocation.
+        timeout (datetime.timedelta, optional): timeout to use for this RPC. If the RPC
```

Comment:
This has to take an int type in order to work with TorchScript.

Reply:
From an API perspective, would it then make sense to have it accept either type, and then in the python code (here) convert to timedelta if we got an int? Or should we do only `int` for the timeout in all cases? cc @mrshenli @pritamdamania87

Reply:
I think we should have only one type. Although, instead of int can we use float similar to what `time.sleep()` does? This way users can specify a much more granular timeout. Internally we can convert to an appropriate int for milliseconds.

Reply:
+1 Agree float style is somewhat more intuitive for timeouts. 
I've seen different apis specify int timeouts in various units - millis, micros, seconds, etc. and can be a real source of bugs.